components/dashboard/src/menu/Menu.tsx

```diff
@@ -98,9 +98,14 @@ export default function Menu() {
                                     </li>
                                 )}
                                 {isGitpodIo() && (
-                                    <li className="cursor-pointer">
-                                        <PillMenuItem name="Feedback" onClick={handleFeedbackFormClick} />
-                                    </li>
+                                    <>
+                                        <li className="cursor-pointer">
+                                            <PillMenuItem name="🪐 CDE Universe ↗" link="https://cdeuniverse.com/" />
```

Comment:
**question**: @selfcontained Any hints on making this open in a new window like `target="_blank"`?